android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthorizationsDao.java

```diff
@@ -89,17 +93,20 @@ public synchronized List<AuthRecord> getAuthorizations(@NonNull IdentityRecord i
         try (final Cursor cursor = super.rawQuery("SELECT " +
                         TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_ID +
                         ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_ISSUED +
-                        ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_PUBLIC_KEY_ID +
+                        ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_ACCOUNT_ID +
                         ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_WALLET_URI_BASE_ID +
                         ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_SCOPE +
                         ", " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_CLUSTER +
-                        ", " + TABLE_PUBLIC_KEYS + '.' + COLUMN_PUBLIC_KEYS_RAW +
-                        ", " + TABLE_PUBLIC_KEYS + '.' + COLUMN_PUBLIC_KEYS_LABEL +
+                        ", " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_PUBLIC_KEY_RAW +
+                        ", " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_LABEL +
+                        ", " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_ICON +
+                        ", " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_CHAINS +
+                        ", " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_FEATURES +
                         ", " + TABLE_WALLET_URI_BASE + '.' + COLUMN_WALLET_URI_BASE_URI +
                         " FROM " + TABLE_AUTHORIZATIONS +
-                        " INNER JOIN " + TABLE_PUBLIC_KEYS +
-                        " ON " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_PUBLIC_KEY_ID +
-                        " = " + TABLE_PUBLIC_KEYS + '.' + COLUMN_PUBLIC_KEYS_ID +
+                        " INNER JOIN " + TABLE_ACCOUNTS +
+                        " ON " + TABLE_AUTHORIZATIONS + '.' + COLUMN_AUTHORIZATIONS_ACCOUNT_ID +
+                        " = " + TABLE_ACCOUNTS + '.' + COLUMN_ACCOUNTS_ID +
```

Comment:
1. `getAuthorizations()` is incorrect if `COLUMN_AUTHORIZATIONS_ACCOUNT_ID` still refers to a `publicKeyId`.

The `INNER_JOIN` will try to match `COLUMN_AUTHORIZATIONS_ACCOUNT_ID == COLUMN_ACCOUNTS_ID`, but `COLUMN_AUTHORIZATIONS_ACCOUNT_ID` columns will all be `publicKeyId` values while its expecting `accountId` values. This results in an empty join.

Reply:
fixed with migration from public_key_id column to account_id